app/javascript/styles/mastodon/components.scss

```diff
@@ -9010,3 +9010,61 @@ noscript {
     }
   }
 }
+
+.notification-list {
+  position: fixed;
+  bottom: 2rem;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+}
+
+.notification-bar {
+  flex: 0 0 auto;
+  position: relative;
+  inset-inline-start: -100%;
```

Comment:
This is not consistent with `left: 1rem` at https://github.com/mastodon/mastodon/pull/25564/files#diff-edd345ee45838ecdc48d8b64ba5f0d4475300e2cc36e696a937fe012a6e025caR9044 and will not get overridden on RTL layouts. Furthermore, this is not consistent with `left: 0` at https://github.com/mastodon/mastodon/pull/25564/files#diff-edd345ee45838ecdc48d8b64ba5f0d4475300e2cc36e696a937fe012a6e025caR9017 and will not be positioned properly on RTL layouts.